specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NspAccessRuleList.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "List NSP Access Rules",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1",
+    "profileName": "profile1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "accessRule1",
+            "id": "/subscriptions/3846cb0f-4afa-47ee-8ea4-1c8449c8c8d9/resourceGroups/Haris-RG/providers/Microsoft.Network/networkSecurityPerimeters/NSP11/profiles/NP2/accessRules/accessRule1",
```

Comment:
NSP11 -> nsp1, 
NP2 -> profile1

Reply:
Also, it might be better to use 'subId' and 'rg1' to keep things uniform